### DISCUSSION TITLE
Admin view - tabs width

### DISCUSSION THREAD

**Discussion by dformdotdk** [joomengine/Joomla-Component-Builder#711] [Q&A]
When we use the normal admin view (not custom), can we affect the tabs width in any way? E.g. "Left in tab" and "Right in tab" both "span6". In some cases, e.g. a span3 / span9 setup is better, so it would be awesome if we could define the span values for each tab when we setup tabs, without having to resort to custom views.

**Comment by dformdotdk** [comment]
An idea: what if we with custom admin tabs could replace a normal tab, if the custom tab used a name identical to a regular tab.
That would make it pretty easy to manipulate the view without having to resort to custom admin views.

**Reply by Llewellynvdm** [reply]
okay I have been working on a path to allow full customization of the admin view. So at this point `drum-roll` we have full customization of layouts in admin views. Yes, it is a pro-feature and by design I need to encourage all to support JCB by donations... and so in a year from it being made available to the Pro-Members... I will push the feature to all.

Yet I can tell you the next step is to move each tap in an admin-view to its own template, and then setup the same feature to customization of all templates, as we already have for the layouts.

So how you @denovoit  as a pro-member can already with the layout achieve the same maneuverability, is to add all your fields to the `fullwidth` option... and then in the layout manage your fields as you like with the pro-feature that I mentioned.

ps. This pro-feature is about to go out as the January release.... so keep a lookout for the next Pro-Member newsletter ;)

**Reply by Llewellynvdm** [reply]
I just looked... my bad. @denovoit look at this pro-release https://vdm.bz/jcb-pro-nov-2020 (the layout feature is explained there) So its already out....

**Reply by dformdotdk** [reply]
I've just seen your video - very powerfull. And indeed calls for a drum-roll 👍 
As you mentioned, a placeholder for the fields, to keep the dynamics in the layout would avoid some headaches I think :-) 

One thing I noticed, is you "only" target layouts. Is there a way to override the template files, e.g. the edit.php file?

The part I would also like to override is this, so I can controll the widths of each column:

	<div class="row-fluid form-horizontal-desktop">
		<div class="span12">
			<div class="row-fluid form-horizontal-desktop">
				<div class="span3">
					<?php echo JLayoutHelper::render('layout.tab_left', $this); ?>
				</div>
				<div class="span9">
					<?php echo JLayoutHelper::render('layout.tab_right', $this); ?>
				</div>
			</div>
		</div>
	</div>

If we add a custom tab, with the same name as the normal tab, we could achieve this if JCB would override/replace the normal tab HTML with the custom tab HTML when the names are identical.
